StRoot/RTS/include/FTP/ftpc_padkey.h

```diff
@@ -0,0 +1,19378 @@
+#ifndef _FTP_PADKEY_H_
```

Comment:
Is this file really used by the stgc and fcs readers?

Reply:
This file conflicts with StRoot/RTS/include/FTP/FTPC_PADKEY.h on case-insensitive systems such as macOS. If the lower-case version `ftpc_padkey.h` is used by the online code, can we remove or rename the upper-case one `FTPC_PADKEY.h`?